TESTING-INSTRUCTIONS.md

```diff
@@ -2,6 +2,23 @@
 
 ## Unreleased
 
+### Store profiler - Added MailPoet to Business Details step  #6503
+
+- Create a brand new site and go to the OBW.
+- In the first OBW step (`Store Details`) set a Country / Region other than `US` (e.g.: Uruguay).
```

Comment:
I merged a PR that narrows down this test instruction, and it can't be a country in this list: `US | BR | FR | ID | GB | DE | VN | CA | PL | MY | AU | NG | GR | BE | PT | DK | SE | JP`
Given Uruguay is still not part of that list, maybe just explicitly say set country to **Uruguay**

Reply:
Good idea! I changed the testing instructions in the commit e8388a8